bitmap-ittybitty/test/app-test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+const app = require('./../lib/app');
+const expect = require('chai').expect;
+const bitmap = fs.readFileSync(__dirname +'/../img/bitmap1.bmp');
+var test = new app(bitmap);
+
+describe('Testing the app module that will parse a buffer', function(){
```

Comment:
Nice work with these--this is a hard project to figure out how to test well. 

That said, a lot of these could be combined. I'd probably have an 'it' that said something like 'should be an object with the correct properties', then done all the expects in that 'it'. As is, it looks like you're doing a whole bunch when really you're just testing that you got anything at all.